### ISSUE TITLE
Instructions mention libffi-devel being needed. Still true?

### ISSUE THREAD

**Issue by jerboaa** [graalvm/mandrel#158]
We have this in our README.md:

```
Mandrel's native-image depends on the following packages:
    glibc-devel
    zlib-devel
    gcc
    libffi-devel
```

Is `libffi-devel` still needed? Isn't that a truffle only dep, which got removed with https://github.com/graalvm/mandrel-packaging/pull/61/files

**Comment by jerboaa** [comment]
@zakkak Thoughts?

**Comment by zakkak** [comment]
@jerboaa you are right:

20.1.0.2.Final fails without `libffi-dev`:
```
$ /opt/jvms/mandrel-java11-20.1.0.2.Final/bin/native-image Hello
[hello:764346]    classlist:   1,185.30 ms,  0.96 GB
[hello:764346]        (cap):     510.83 ms,  0.96 GB
[hello:764346]        setup:   1,042.90 ms,  0.96 GB
Error: Error compiling query code (in /tmp/SVM-17124881261326283124/LibFFIHeaderDirectives.c). Compiler command '/usr/bin/gcc -I/opt/jvms/mandrel-java11-20.1.0.2.Final/lib/svm/clibraries/linux-amd64/include -o /tmp/SVM-17124881261326283124/LibFFIHeaderDirectives /tmp/SVM-17124881261326283124/LibFFIHeaderDirectives.c ' output included error: /opt/jvms/mandrel-java11-20.1.0.2.Final/lib/svm/clibraries/linux-amd64/include/svm_libffi.h:25:10: fatal error: ffi.h: No such file or directory
Error: Use -H:+ReportExceptionStackTraces to print stacktrace of underlying exception
Error: Image build request failed with exit status 1
```

While 20.1.0.3.Final works fine:
```
$ /opt/jvms/mandrel-java11-20.1.0.3.Final/bin/native-image Hello                                                  
[hello:764136]    classlist:   1,043.02 ms,  0.96 GB
[hello:764136]        (cap):     479.02 ms,  0.96 GB
[hello:764136]        setup:   1,801.94 ms,  0.96 GB
[hello:764136]     (clinit):     202.70 ms,  1.70 GB
[hello:764136]   (typeflow):   3,907.88 ms,  1.70 GB
[hello:764136]    (objects):   4,281.54 ms,  1.70 GB
[hello:764136]   (features):     245.76 ms,  1.70 GB
[hello:764136]     analysis:   8,862.83 ms,  1.70 GB
[hello:764136]     universe:     298.16 ms,  1.70 GB
[hello:764136]      (parse):     825.20 ms,  1.70 GB
[hello:764136]     (inline):   1,368.85 ms,  1.70 GB
[hello:764136]    (compile):   6,337.25 ms,  2.32 GB
[hello:764136]      compile:   8,960.94 ms,  2.32 GB
[hello:764136]        image:   1,185.99 ms,  2.32 GB
[hello:764136]        write:     153.22 ms,  2.32 GB
[hello:764136]      [total]:  22,514.35 ms,  2.32 GB
```

**Comment by jerboaa** [comment]
> @jerboaa you are right:
> 
> 20.1.0.2.Final fails without `libffi-dev`:
> 
> ```
> $ /opt/jvms/mandrel-java11-20.1.0.2.Final/bin/native-image Hello
> [hello:764346]    classlist:   1,185.30 ms,  0.96 GB
> [hello:764346]        (cap):     510.83 ms,  0.96 GB
> [hello:764346]        setup:   1,042.90 ms,  0.96 GB
> Error: Error compiling query code (in /tmp/SVM-17124881261326283124/LibFFIHeaderDirectives.c). Compiler command '/usr/bin/gcc -I/opt/jvms/mandrel-java11-20.1.0.2.Final/lib/svm/clibraries/linux-amd64/include -o /tmp/SVM-17124881261326283124/LibFFIHeaderDirectives /tmp/SVM-17124881261326283124/LibFFIHeaderDirectives.c ' output included error: /opt/jvms/mandrel-java11-20.1.0.2.Final/lib/svm/clibraries/linux-amd64/include/svm_libffi.h:25:10: fatal error: ffi.h: No such file or directory
> Error: Use -H:+ReportExceptionStackTraces to print stacktrace of underlying exception
> Error: Image build request failed with exit status 1
> ```
> 
> While 20.1.0.3.Final works fine:
> 
> ```
> $ /opt/jvms/mandrel-java11-20.1.0.3.Final/bin/native-image Hello                                                  
> [hello:764136]    classlist:   1,043.02 ms,  0.96 GB
> [hello:764136]        (cap):     479.02 ms,  0.96 GB
> [hello:764136]        setup:   1,801.94 ms,  0.96 GB
> [hello:764136]     (clinit):     202.70 ms,  1.70 GB
> [hello:764136]   (typeflow):   3,907.88 ms,  1.70 GB
> [hello:764136]    (objects):   4,281.54 ms,  1.70 GB
> [hello:764136]   (features):     245.76 ms,  1.70 GB
> [hello:764136]     analysis:   8,862.83 ms,  1.70 GB
> [hello:764136]     universe:     298.16 ms,  1.70 GB
> [hello:764136]      (parse):     825.20 ms,  1.70 GB
> [hello:764136]     (inline):   1,368.85 ms,  1.70 GB
> [hello:764136]    (compile):   6,337.25 ms,  2.32 GB
> [hello:764136]      compile:   8,960.94 ms,  2.32 GB
> [hello:764136]        image:   1,185.99 ms,  2.32 GB
> [hello:764136]        write:     153.22 ms,  2.32 GB
> [hello:764136]      [total]:  22,514.35 ms,  2.32 GB
> ```

Thanks for clarifying.
